### PR TITLE
iOS build

### PR DESCRIPTION
**Pull request by 1Conan**
There's a warning about missing symbols for libv8_base_without_compiler.a when building the dSYM file in the framework.

##### Checklist

- [ ] tests and/or benchmarks are included
- [x] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)

### CODE REVIEW

**Comment by staltz** [comment]
This is a lot of progress! Thanks! I'll have to confirm on my side whether it's working before merging. It may take some time, but I'll get to it.

**Comment by 1Conan** [comment]
Sure thing :)

Only thing missing here are unit tests which I'm not too familiar with.

**Comment by KishanBagaria** [comment]
Can confirm, it's working well so far, even with native modules. 

I wonder if it's trivial to update to the latest Node.js v16.14.0?

File: tools/ios_framework_prepare.sh [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        --with-intl=none \
+        --cross-compiling \
+        --enable-static \
+        --openssl-no-asm \
+        --v8-options=--jitless \
+        --without-node-code-cache \
+        --without-node-snapshot \
+        --v8-disable-webassembly
+
+    make -j$(getconf _NPROCESSOR_ONLN)
```
Comment (bummoblizard) [inline tools/ios_framework_prepare.sh:70]: There is

**Comment by KishanBagaria** [comment]
@1Conan sent me the compiled .framework. I've converted it to an xcframework with `xcodebuild -create-xcframework -framework NodeMobile.framework -output NodeMobile.xcframework` and uploaded here: https://texts.com/dev/NodeMobile16.xcframework.zip (sha256: 19f8f624d68b8de5a570fe0f512c4d08b37d290e9a4bfb97f25ea135e446ad05)

**Comment by bummoblizard** [comment]
> @1Conan sent me the compiled .framework. I've converted it to an xcframework with `xcodebuild -create-xcframework -framework NodeMobile.framework -output NodeMobile.xcframework` and uploaded here: https://texts.com/dev/NodeMobile16.xcframework.zip (sha256: 19f8f624d68b8de5a570fe0f512c4d08b37d290e9a4bfb97f25ea135e446ad05)

Great! Thank you. That will do it for now but I think changes probably still need to be made before merging.

**Comment by 1Conan** [comment]
> `NodeMobile.xcodeproj` seems to be missing. How did you manage to build the framework? @KishanBagaria

I forgot that it's gitignore-d by default I'll commit it later on.

**Comment by bummoblizard** [comment]
> > `NodeMobile.xcodeproj` seems to be missing. How did you manage to build the framework? @KishanBagaria
> 
> I forgot that it's gitignore-d by default I'll commit it later on.

Hey @1Conan. It would be really appreciated if you can include internationalization support as well. Drop me an email at ken.chung@thebaselab.com if you'd like bounty.

**Comment by 1Conan** [comment]
@bummoblizard sent you an email about it :)

**Comment by staltz** [comment]
Hey, thanks for your PR, it provided some inspiration and guidance. However, I have now merged #9 which updated nodejs-mobile to 16.17.1 for both Android and iOS. Can we close this PR?

**Pull request closed without merging**